README.md

```diff
@@ -1 +1,22 @@
-# one-click-installers
+# GleSYS one-click installers [![License: CC0-1.0](https://img.shields.io/badge/License-CC0%201.0-lightgrey.svg)](http://creativecommons.org/publicdomain/zero/1.0/)
+> The scripts that powers the one-click installers on GleSYS Cloud (cloud.glesys.com).
+
+## Guidelines
+
+1. The one-click installer should be placed in a folder which is named after a rundown of what it does. Eg. `wordpress-nginx-mariadb`.
```

Comment:
WIll there be such a directory soon? I do think I understand the sentinement but having a actual example to look at helps.

Reply:
I was thinking that it would be clear if you look at the existing folders in the repo. But it might be smart to maybe create a `example-folder` folder?